PublishPackageManagementsUpdates.yml

```
@@ -55,6 +58,129 @@
         echo "Versions are the same. Skipping PyPI upload."
       fi
+    name: PypiPublishStep
+
+  - job: CreateGitHubRelease
+    displayName: 'Create GitHub Release'
+    dependsOn: CheckPypiVersionBeforePublishing
+    pool:
+    vmImage: ubuntu-latest
+  variables:
+    RELEASE_VERSION: $[ dependencies.CheckPypiVersionBeforePublishing.outputs['VersionExtractionStep.packageVersion'] ]
+  steps:
```

YAMLlint (1.37.1)

[error] duplication of key "steps" in mapping

(key-duplicates)